source/includes/open-new-connection.rst

```diff
@@ -1,3 +1,7 @@
-In the bottom panel of the :guilabel:`Connections Sidebar`,
-click the :icon-lg:`Plus` icon to open the :guilabel:`New Connection`
-modal. 
\ No newline at end of file
+In the bottom panel of the :guilabel:`Connections Sidebar`, click
```

Comment:
*Question* 

Is this file included anywhere? I did a quick search in master and dont see it included and dont see it in the PR, could we also remove this?

<img width="1122" alt="Screenshot 2024-08-14 at 2 09 39 PM" src="https://github.com/user-attachments/assets/c3e9b19d-c0ba-4458-8400-105df78b96ee">

Reply:
Done!

Reply:
This file is added in the multiple-connections branch and is used in the advanced connection options pages (just added the staging link). It's not currently on master but will be once the feature branch is merged.